views/pages/index.ejs

```diff
@@ -8,5 +8,6 @@
         </div>  
     </div>
     <h1>Explanation of the site functions would go here.</h1>
+<%- include('../partials/footer')-%>
 </body>
```

Comment:
@ZM202 Lets remove the closing body and the closing html tag and add them into the footer.ejs file.

Reply:
All set